Docs/Workflows.md

```diff
@@ -99,3 +99,24 @@ jobs:
 If this is for a submodule of [Lombiq's Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/), the `repo` input can be omitted, because the above is its default value. Otherwise, use your parent repository's address in the `{owner}/{repo_name}` format.
 
 Refer to [Github Actions reusable workflows](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#overview) for more information.
+
+## Jira issue-creating for community activities workflow
+
+Creates Jira issues for community activities happening on GitHub, like issues, discussions, pull requests being opened.
```

Comment:
We should explain what makes the activities "community". Seems to me the distinction only exists for pull requests anyway. In other words this creates Jira issues for any issue or discussion and for pull requests that don't already contain a Jira issue code.

Reply:
It's just that activities on a public GH project for OSS software = community activity. For us two there's indeed the connotation that it's only "community" activity if not done by a Lombiq team member. I modified the description so it's sufficiently clear for anybody outside of Lombiq too.